ArduPlane/Log.cpp

```diff
@@ -232,7 +234,9 @@ void Plane::Log_Write_AETR()
         ,throttle : SRV_Channels::get_output_scaled(SRV_Channel::k_throttle)
         ,rudder   : SRV_Channels::get_output_scaled(SRV_Channel::k_rudder)
         ,flap     : SRV_Channels::get_slew_limited_output_scaled(SRV_Channel::k_flap_auto)
-        ,speed_scaler : get_speed_scaler(),
+        ,speed_scaler_x : get_rpy_speed_scaler().x
+        ,speed_scaler_y : get_rpy_speed_scaler().y
+        ,speed_scaler_z : get_rpy_speed_scaler().z
```

Comment:
Not really a fan of logging all three values here since there the same number multiplied by a constant..

Reply:
Thats not correct. However we could log the inflow factor and save one value given that scaling_speed = airspeed * (1 +2*a*K) where 'a' is inflow factor and 'K' is the PROP_*_COMP value. However for tuning the speed scaler is more useful